examples/2.0.0/aws-sns-publish.yml

```diff
@@ -0,0 +1,80 @@
+asyncapi: 2.0.0
+id: 'https://github.com/asyncapi/asyncapi/blob/master/examples/2.0.0/aws-sns-publish.yml'
+info:
+  title: 'Simple AWS SNS Publish specification'
+  description: |
+    Specification for a service that publishes various "hello world"
+    type messages to the helloWorld SNS topic
+  version: '0.1.0'
+servers:
+  development:
+    # This URL doesn't really describe a development server, but it does describe the
+    # top-level URL used to subscribe to messages from a given AWS region
+    url: 'https://sns.us-east-1.amazonaws.com/'
+    description: 'Development/test account for message publishing'
+    protocol: 'sns'
+    protocolVersion: '2010-03-31'
+    bindings:
+      # AccountID for this environment
+      awsAccountID: "123456789012"
+  test:
+    url: 'https://sns.us-east-1.amazonaws.com/'
+    description: 'Development/test account for message publishing'
+    protocol: 'sns'
+    protocolVersion: '2010-03-31'
+    bindings:
+      awsAccountID: '123456789012'
+  production:
+    url: 'https://sns.us-east-1.amazonaws.com/'
+    description: 'Production account for message publishing'
+    protocol: 'sns'
+    protocolVersion: '2010-03-31'
+    bindings:
+      awsAccountID: '567891011121'
+channels:
+  helloWorld:
+    description: 'Topic used for hello world and similar messages'
+    bindings:
+      $ref: '#/components/channelBindings/sns/helloWorld'
+    subscribe:
+      message:
+        oneOf:
+          - $ref: '#/components/messages/helloWorldMessage'
+          - $ref: '#/components/messages/goodNightWorldMessage'
+components:
+  channelBindings:
+    sns:
+      helloWorld:
+        development:
+          topicArn: 'arn:aws:sns:us-east-1:123456789012:helloWorld-dev'
```

Comment:
If we put the region information suggested above, then we would not need this, right? The generator (or other software) could come up with the topicArn string by joining all the information: `arn:aws:sns:` + region from the server + `:` + account from the server + `:` + channel name.

Let me know if this makes sense. I'm trying to decouple the binding information from the server definitions.